packages/Card/src/components/Avatar/helpers.js

```diff
@@ -0,0 +1,27 @@
+import Tokens from "@paprika/tokens";
+
+export default function calculateAvatarColors(text) {
```

Comment:
I might prefer `getAvatarColors()` but 🤷‍♂️ 
I guess this will get migrated / deprecated anyway when we create a `@paprika/avatar` component?

Reply:
It should get migrated -- not deprecated! 😱